app/scenes/thread/thread_container.js

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {bindActionCreators} from 'redux';
+import {createSelector} from 'reselect';
+
+import {handleCommentDraftChanged} from 'app/actions/views/thread';
+
+import {getAllPosts} from 'service/selectors/entities/posts';
+import {getTheme} from 'service/selectors/entities/preferences';
+
+import navigationSceneConnect from '../navigationSceneConnect';
+import Thread from './thread';
+
+export function makeGetPostsForThread() {
+    return createSelector(
```

Comment:
I can change that. I think it had some component-specific stuff in it at one point that I removed like the one for the `ChannelPostList`.